subprojects/common/report-viewer/src/main/kotlin/com/avito/report/model/RunId.kt

```diff
@@ -3,22 +3,43 @@ package com.avito.report.model
 import com.avito.android.Result
 
 /**
- * Универсальный runId для идентификации сьюта тестов
- * - build config teamcity домешан, чтобы отличать прогон на одном API на develop
- *  (там так же как в релизе нет targetBranch) от buildRelease
+ * Universal identifier for test suits
+ * @param prefix optional prefix
+ * @param commitHash unique SHA identifier of commit
+ * @param buildTypeId is used to distinguish runs with the same API on `develop` branch
  */
-data class RunId(val commitHash: String, val buildTypeId: String) {
+data class RunId(
+    val prefix: String = "",
+    val commitHash: String,
+    val buildTypeId: String
+) {
 
-    override fun toString(): String = "$commitHash$DELIMITER$buildTypeId"
+    init {
+        require(!prefix.contains(DELIMITER)) { "prefix could not contain '$DELIMITER' character" }
+        require(!commitHash.contains(DELIMITER)) { "commitHash could not contain '$DELIMITER' character" }
+        require(!buildTypeId.contains(DELIMITER)) { "buildTypeId could not contain '$DELIMITER' character" }
+    }
+
+    override fun toString(): String = when {
+        prefix.isBlank() -> "$commitHash$DELIMITER$buildTypeId"
+        else -> "$prefix$DELIMITER$commitHash$DELIMITER$buildTypeId"
+    }
 
     companion object {
 
-        private const val DELIMITER = '.'
```

Comment:
Why do we need that change?

Reply:
Doesn't matter now. There were difficulties with parsing RunId from string, but it's used nowhere. Brought back the old value.